client/components/Editor/index.jsx

```diff
@@ -146,8 +175,33 @@ class Editor extends Component {
                         </Menu.Item>
                     </Menu.Menu>
                 </Menu>
+                <Segment attached="bottom">
+                    {this.state.scenarioId !== 'new' && (
+                        <Menu icon>
+                            <Menu.Item
+                                name="save-scenario"
+                                onClick={this.onClickScenarioAction}
+                            >
+                                <Icon name="save" />
+                            </Menu.Item>
 
-                <div>{this.state.tabs[this.state.activeTab]}</div>
+                            <Menu.Item
+                                name="delete-scenario"
+                                onClick={this.onClickScenarioAction}
```

Comment:
This ends up being a no-op because `ConfirmableDeleteButton` handles everything. I found that I needed to include an `onClick` property in order to get the right cursor and pointer behavior from the button